modules/statics/src/coins.ts

```diff
@@ -871,6 +871,127 @@ export const coins = CoinMap.fromCoins([
     undefined,
     Networks.test.goerli
   ),
+  terc20(
+    'gtaave18dp',
+    'Goerli Test AAVE Token 18 Decimals',
+    18,
+    '0x631d5e3c45a459e8f98b9d6a2734fce7b051f845',
+    UnderlyingAsset.GTAAVE18DP,
+    undefined,
+    undefined,
+    undefined,
```

Comment:
why is this undefined?

Reply:
![image](https://user-images.githubusercontent.com/85512331/139726952-1cb6270d-6343-478b-93d9-c2b66d50b3b0.png)
because I use this function and I need to pass the last parameter (network) and the other 3 (features, prefix, suffix) I don't use them